gallery/how_to/work_with_microtvm/micro_mlperftiny.py

```diff
@@ -0,0 +1,285 @@
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+.. _tutorial-micro-MLPerfTiny:
+
+Create Your MLPerfTiny Submission with microTVM
+===========================
+**Authors**:
+`Mehrdad Hessar <https://github.com/mehrdadh>`_
+
+This tutorial is showcasing building an MLPerTiny submission using microTVM. This
+tutorial shows the steps to import a TFLite model from MLPerfTiny benchmark models,
+compile it with TVM and generate a Zephyr project which can be flashed to a Zephyr
+supported board to benchmark the model using EEMBC runner.
+
+Install CMSIS-NN only if you are interested to generate this submission
+using CMSIS-NN code generator.
+"""
+
+######################################################################
+#
+#     .. include:: ../../../../gallery/how_to/work_with_microtvm/install_dependencies.rst
+#
+
+import os
+import pathlib
+import tarfile
+import tempfile
+import shutil
+
+######################################################################
+#
+#     .. include:: ../../../../gallery/how_to/work_with_microtvm/install_zephyr.rst
+#
+
+######################################################################
+#
+#     .. include:: ../../../../gallery/how_to/work_with_microtvm/install_cmsis.rst
+#
+
+######################################################################
+# Import Python dependencies
+# -------------------------------
+#
+import tensorflow as tf
+import numpy as np
+
+import tvm
+from tvm import relay
+from tvm.relay.backend import Executor, Runtime
+from tvm.contrib.download import download_testdata
+from tvm.micro import export_model_library_format
+from tvm.micro.model_library_format import generate_c_interface_header
+from tvm.micro.testing.utils import (
+    create_header_file,
+    mlf_extract_workspace_size_bytes,
+)
+
+######################################################################
+# Import Visual Wake Word Model
+# --------------------------------------------------------------------
+#
+# To begin with, download and import Visual Wake Word (VWW) TFLite model from MLPerfTiny.
+# This model is originally from `MLPerf Tiny repository <https://github.com/mlcommons/tiny>`_.
+# We also capture metadata information from the TFLite model such as input/output name,
+# quantization parameters and etc which will be used in following steps.
+#
+# We use indexing for various models to build the submission. The indices are defined as bellow.
+# To build another model, you need to update the model URL, the short name and index number.
+#   Keyword Spotting(KWS)       1
+#   Visual Wake Word(VWW)       2
+#   Anomaly Detection(AD)       3
+#   Image Classification(IC)    4
+#
+# If you like to build the submission with CMSIS-NN, modify USE_CMSIS variable.
+#
+
+MODEL_URL = "https://github.com/mlcommons/tiny/raw/bceb91c5ad2e2deb295547d81505721d3a87d578/benchmark/training/visual_wake_words/trained_models/vww_96_int8.tflite"
```

Comment:
should we include other models, since we reference them elsewhere?

Reply:
I'm with @mehrdadh - I worry having multiple URLS might be confusing. Adding a micro API call would be a good way to clean this up in a future PR.

Reply:
Can this line (101) be pleased changed as follows?

MODEL_URL = os.environ.get("MODEL_URL", "https://github.com/mlcommons/tiny/raw/bceb91c5ad2e2deb295547d81505721d3a87d578/benchmark/training/visual_wake_words/trained_models/vww_96_int8.tflite")

This can give flexibility to the user to change the url without having to modify the code.

Reply:
@arjunsuresh  Unfortunately that's not the only change that is required to use a different model in this tutorial. We could change this tutorial in a follow up PR to get all the model URL, name and index from test APIs in TVM